lib/src/main/java/org/exoplatform/social/addons/updater/ActivityMigrationService.java

```diff
@@ -215,23 +208,14 @@ private long migrateSpaceActivities() throws Exception {
           Node node = null;
 
           while(it.hasNext()) {
-            if (forceStop) {
+            if (forkStop) {
```

Comment:
What is this attribute for ? is it to force stop or to fork ?
and where its value could be changed ?